README.md

```diff
@@ -1 +1,207 @@
+      "badge_url": "https://github.com/ansible-community/ppa/workflows/ansible-core/badge.svg"
+    },
+    {
+      "id": 13724938,
+      "node_id": "W_kwDOFNAryM4A0W0K",
+      "name": "latest builds",
+      "path": ".github/workflows/latest_builds.yml",
+      "state": "active",
+      "created_at": "2021-09-30T21:15:42.000Z",
+      "updated_at": "2021-12-22T19:23:39.000Z",
+      "url": "https://api.github.com/repos/ansible-community/ppa/actions/workflows/13724938",
+      "html_url": "https://github.com/ansible-community/ppa/blob/main/.github/workflows/latest_builds.yml",
+      "badge_url": "https://github.com/ansible-community/ppa/workflows/latest%20builds/badge.svg"
+    }
+  ]
+}
+
+# Then (for example) to generate a new 5.3.0-2ppa~impish build for the ansible-5 PPA, you could make the following POST request. Note the 7276925 workflow ID used from the previous request to denote the ansible (.github/workflows/ansible.yml) workflow.
+
+% curl --location --request POST 'https://api.github.com/repos/ansible-community/ppa/actions/workflows/7276925/dispatches' \
+--header 'Authorization: token ghp_ACTUAL_GITHUB_PAT_TOKEN_GOES_HERE' \
+--header 'Content-Type: application/json' \
+--data-raw '    {
+        "ref": "ansible-5",
+        "inputs": {
+            "DEB_DIST": "impish",
+            "DEB_VERSION": "5.3.0",
+            "DEB_RELEASE": "2ppa",
+            "LAUNCHPAD_PROJECT": "~ansible",
+            "LAUNCHPAD_PPA": "ansible-5"
+        }
+    }'
+```
+
+### Post DATA Reference
+
+| Name | Description |
+| --- | --- |
+| `ref` | Branch to use <br> Ex: `ansible-5` References https://github.com/ansible-community/ppa/tree/ansible-5 |
+| `DEB_DIST` | Ubuntu short code name <br> Ex: `impish` |
+| `DEB_VERSION` | Version <br> Ex: `5.3.0` |
+| `DEB_RELEASE` | Deb release number <br> Ex: `2ppa` |
+| `LAUNCHPAD_PROJECT` | Name of the Launchpad Project to use <br> Ex: `~ansible` would reference https://launchpad.net/~ansible |
+| `LAUNCHPAD_PPA` | Name of the PPA under the Launchpad Project to use <br> Ex: `ansible-5` with the `LAUNCHPAD_PROJECT` value of `~ansible` would reference https://launchpad.net/~ansible/+archive/ubuntu/ansible-5 |
+
+## Repo Requirements
+
+### Secrets
+
+| Name | Description | Example |
+| --- | --- | --- |
+| `DEBSIGN_KEYID` | Deb signing key ID | `4697549E1287BC6A1A481B98DE6FE1F1C7DBF004` |
+| `LAUNCHPAD_PROJECT` | Launchpad project name | `~ansible` |
+| `PAT` | GitHub PAT for starting additional workflows | `ghp_ACTUAL_GITHUB_PAT_TOKEN_GOES_HERE` |
+| `SIGNING_KEY` | Armored signing key | <pre>-----BEGIN PGP PRIVATE KEY BLOCK-----<br/><br/>xcaGBGFLvFkBEADQbd7DocOo9XzMo5PD<br/>RX4Nxw4UHWJGjzdBZgmKzk+vLuTR+Cr8<br/>...<br/>tTtohAGXcR9EYhHemdVrew==<br/>=4DaA<br/>-----END PGP PRIVATE KEY BLOCK-----</pre> |
+| `SIGNING_OWNERTRUST` | Import-able ownertrust string | `4697549E1287BC6A1A481B98DE6FE1F1C7DBF004:6:` |
+| `SIGNING_PASSPHRASE` | Signing key passphrase | `PASSPHRASE_USED_FOR_SIGNING_KEY` |
```

Comment:
At the end of the README could you please add some details about launchpad

* Link to the PPA
* How do we test? Is there a testing PPA (like https://test.pypi.org/)

Reply:
it's not at the end, but I think I've got all of this covered now, including a high level summary of creating a test environment (aka using your own launchpad project and repo fork). please let me know or resolve. thanks!